app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,7 @@
  *= require_tree .
  *= require_self
  */
+
+#petition_form {
+    float: right;
```

Comment:
I'm going to say 4 for the moment because that's what our external CSS files (like bootstrap) seem to use. If we decide we want to do a mass migration later on, I'm OK with that.